src/UniversalRewardsDistributor.sol

```diff
@@ -122,9 +122,9 @@ contract UniversalRewardsDistributor is IUniversalRewardsDistributor {
             ErrorsLib.INVALID_PROOF_OR_EXPIRED
         );
 
-        amount = claimable - claimed[account][reward];
+        require(claimable > claimed[account][reward], ErrorsLib.CLAIMABLE_TOO_LOW);
 
-        require(amount > 0, ErrorsLib.ALREADY_CLAIMED);
```

Comment:
I understand that now we have the other `requirement` that already covers this scenario, but now when the user tries to claim the reward that has been already claimed it will throw an error that is not related to this problem.

One thing is when `claimable < claimed[account][reward]`, this means that there's a problem in the `root`

Another thing is when `claimable == claimed[account][reward]`, this means that the user has already claimed the reward and the root has not been updated, but there's no real issue with the current root.

Reply:
We have decided to merge the two scenarios in one: `CLAIMABLE_TOO_LOW` that can handle both. The message is explicit and we assume the Merkle tree specs are respected (detailed into the readme)